### PR TITLE
Rozie100/les

### PR DESCRIPTION
**Pull request by Rozie100**
LES-Model formula is added under documentation.

Note that in the literature, the LES is represented in the strong format of Navier–Stokes equations only.

### CODE REVIEW

**Review by jikembo — APPROVED** [review]
Looks fine!

File: docs/content/formulations/compressibleFlow/LES-Model.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: LES Model
+parent: Compressible Flow 
+grand_parent: Flow Formulations
```
Comment (mmcgurn) [inline docs/content/formulations/compressibleFlow/LES-Model.md:5]: Can you change this to :

```
parent: Compressible Flow
grand_parent: Formulations
```

**Review by mmcgurn — CHANGES_REQUESTED** [review]
Looks Good, I just made one small request.

**Review by Rozie100 — COMMENTED** [review]
Fixed!

File: docs/content/formulations/compressibleFlow/LES-Model.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Large-eddy Simulation of Compressible Flows
+
+### The filtered compressible Navier-Stokes equations can be expressed as (Hirch [1]):
+
+$$\begin{equation}\frac{\partial \bar{\rho}}{\partial t} + \frac{\partial(\bar{\rho} \tilde{u}_{j})}{\partial x_{j}} = 0\end{equation}$$
+
+$$\begin{equation}\frac{\partial (\bar{\rho} \tilde{u}_{i})}{\partial t} + \frac{\partial(\bar{\rho} \tilde{u}_{i}\tilde{u}_{j})}{\partial x_{j}} = -\frac{\partial \bar{p}}{\partial x_{i}} + \frac{\partial \mu \tilde{S}_{ij}}{\partial x_{j}} -
+ \frac{\partial \tau_{ij}}{\partial x_{j}}\end{equation}$$
+
+$$\begin{equation}\frac{\partial (\bar{\rho} \tilde{e})}{\partial t} + (\bar{\rho} \tilde{e}+\bar{p})\frac{\partial \tilde{u}_{j}}{\partial x_{j}} = \frac{\partial(\mu \tilde{S}_{ij}-\tau_{ij})\tilde{u}_{i}}{\partial x_{j}} + \frac{\partial }{\partial x_{j}}\left(\lambda\frac{\partial \tilde{T}}{\partial x_{j}}\right) - \frac{\partial{ q_{j}}}{\partial x_{j}}+ \bar{\dot{\omega}}_T\end{equation}$$
```
Comment (mmcgurn) [inline docs/content/formulations/compressibleFlow/LES-Model.md:17]: I pulled the changes and rended the page.  It looks like we are missing some double $$ around some of the items.  Would you mind making sure they are all double $$.  Thanks!

image

**Review by mmcgurn — CHANGES_REQUESTED** [review]

**Review by Rozie100 — COMMENTED** [review]
Can you check if the last line is rendered correctly?
" is the grid size and the default model coefficients are $$ C_k=0.094, C_\epsilon =1.048$$.

**Review by mmcgurn — APPROVED** [review]
Thanks for making those changes!  It looks good.

**Pull request merged**